manifests/xbuild.pp

```diff
@@ -0,0 +1,14 @@
+# Install cross-building dependencies
+class poudriere::xbuild {
```

Comment:
is this class supposed to be called directly or can we add `assert_private()`? Please also add puppet-strings style documentation for the new class ( a `@summary` line)

Reply:
My feeling is that users who do not rely on `poudriere::env` can benefit from this class by including it in their profile.